packages/tc-api-rest-handlers/lib/separate-documents-from-body.js

```diff
@@ -8,14 +7,9 @@ const separateDocsFromBody = (nodeType, body = {}) => {
 
 	Object.entries(body).forEach(([key, value]) => {
 		if (key.charAt(0) !== '!' && properties[key].type === 'Document') {
-			// We should check the value is empty only when type is Document
-			// because other type will be passed as boolean, number, etc...
-			// then _isEmpty() will returns true
-			if (!_isEmpty(value)) {
-				bodyDocuments[key] = body[key];
-			}
+			bodyDocuments[key] = value;
```

Comment:
Should there be a check for '!' and type = Document?  Currently it just falls through to the else.

Reply:
Starting with `!` is only valid for relationships, so having both together isn't possible. Happy to add a comment though